src/addressResolvers/ens.ts

```diff
@@ -66,7 +66,7 @@ export async function resolveNames(handles: Handle[]): Promise<Record<Handle, Ad
         data: { domains: items }
       }
     } = await graphQlCall(
-      'https://api.thegraph.com/subgraphs/name/ensdomains/ens',
+      'https://subgrapher.snapshot.org/subgraph/arbitrum/5XqPmWe6gjyrJtFn9cLy237i4cWw2j9HcUJEXsP5qGtH',
```

Comment:
Subgraph is on arbitrum ?

Reply:
Yep, https://thegraph.com/explorer/subgraphs/5XqPmWe6gjyrJtFn9cLy237i4cWw2j9HcUJEXsP5qGtH?chain=arbitrum-one deployed on arbitrum but indexing on mainnet